### PR TITLE
Label dgraph_export volume

### PR DESCRIPTION
**Pull request by ghost**
Since docker bake doesn't support creating volumes at the moment, we
are labeling the volume inside of docker-compose. This will enable
better filtering in Make commands and/or Nomad

### What changes does this PR make to Grapl? Why?
This adds basic labels to the dgraph_export volume. This will make it easier to filter and work with in the future.

### How were these changes tested?
Remove the dgraph volume, then recreate the volume via make up

### CODE REVIEW

**Review by wimax-grapl — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -4,8 +4,16 @@
 # 84xx - debugger ports (see vsc_debugger.py)
 
 version: "3.8"
+# docker-bake doesn't support volume creation, so we define and label them here
 volumes:
   dgraph_export:
+    # Labels are only applied to newly created volumes. Existing volumes will NOT have labels updated
+    labels:
+      "org.opencontainers.image.title": "dgraph_export"
+      "org.opencontainers.image.vendor": "Grapl, Inc."
```
Comment (inickles-grapl) [inline docker-compose.yml:13]: Note these are "image" label keys, prob not what we want for volumes.

File: docker-compose.yml [hunk 2]
```diff
@@ -4,8 +4,16 @@
...
+      # This label is for backwards compatibility
+      "com.docker.compose.project": "grapl"
```
Comment (christophermaier) [inline docker-compose.yml:15]: Backwards compatibility with what?

File: docker-compose.yml [hunk 3]
```diff
@@ -4,8 +4,16 @@
 # 84xx - debugger ports (see vsc_debugger.py)
 
 version: "3.8"
+# docker-bake doesn't support volume creation, so we define and label them here
```
Comment (christophermaier) [inline docker-compose.yml:7]: I don't think this comment is really necessary. `docker buildx bake` is about building images; volumes don't apply to that usecase.

File: docker-compose.yml [hunk 4]
```diff
@@ -4,8 +4,16 @@
...
 volumes:
   dgraph_export:
+    # Labels are only applied to newly created volumes. Existing volumes will NOT have labels updated
+    labels:
+      "org.opencontainers.image.title": "dgraph_export"
+      "org.opencontainers.image.vendor": "Grapl, Inc."
+      # This label is for backwards compatibility
+      "com.docker.compose.project": "grapl"
+      maintainer: "Dgraph Labs <contact@dgraph.io>"
```
Comment (christophermaier) [inline docker-compose.yml:16]: Why is this here? Dgraph doesn't maintain our volumes 🙃
